android-json-form-wizard/src/main/java/com/vijay/jsonwizard/customviews/NativeEditText.java

```diff
@@ -156,7 +156,7 @@ public boolean validate() {
         }
 
         CharSequence text = getText();
-        boolean isEmpty = text.length() == 0;
+        boolean isEmpty = text.toString().trim().length() == 0;
```

Comment:
How about using the `StringUtils#isEmpty `method from the apache common library?

Reply:
You can have it check for `isBlank` which will look for white spaces so you do not have to trim

Reply:
Replaced with `StringUtils.isBlank(text)`